Gemfile

```diff
@@ -51,9 +51,9 @@ gem 'rails_12factor', group: :production
 gem 'devise'
 
 #include bootstrap
-gem 'bootstrap-sass', '~> 3.3.5.1'
-gem 'sprockets', '2.11.0'
+gem 'bootstrap-sass', '~> 3.3.6'
+gem 'sprockets', '>= 3.2'
 
-# OmniAuth Authentication 
+# OmniAuth Authentication
 gem 'omniauth-google-oauth2'
 gem 'omniauth-facebook'
```

Comment:
This commit was to resolve the merge conflict I had. I updated the Gemfile in this branch to reflect the latest Gemfile update in #2. I believe the build failed because Codeship has been setup with RSpec. This is the error in Codeship: 
**rspec-core is not part of the bundle. Add it to Gemfile. (Gem::LoadError)**

I'm going to wait until #5 is merged before adding another commit for this.